frontend/styles.css

```diff
@@ -0,0 +1,18 @@
+h1 {
+    text-align: center;
+    border: 2px solid grey;
+    padding: 20px;
+    color: blue;
+    font-size: 500%;
+}
+
+img {
+    width: 50%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+h3 {
+    text-align: center;
+}
```

Comment:
We might talk about CSS best practices in the future, but this looks totally fine for now!